advanced/Scripts/updatecheck.sh

```diff
@@ -41,54 +41,78 @@ function get_local_version() {
 # shellcheck disable=SC1091
 . /etc/pihole/setupVars.conf
 
+utilsfile="/opt/pihole/utils.sh"
+source "${utilsfile}"
```

Comment:
ShellCheck is still complaining.
Suggestion:

```suggestion
utilsfile="/opt/pihole/utils.sh"
# shellcheck disable=SC1091
. "${utilsfile}"
```

Reply:
Huh, I don't know how this came to pass, I definitely added the comment to disable 1091! Nevermind... it will be there in next push